Data/core_rulebook/cr_abilities.lua

```diff
@@ -4360,6 +4380,11 @@ ModifyAbility({
         "ShieldBashingDieSizeStep",
       },
       Formula=Formula("4"),
+      Type={
+        Name="ShieldSize",
+        Replace=false,
```

Comment:
This "Replace" "Stack" stuff seems like false could be inferred, since stacking and replacing is quite rare.